cmd/config-migrater/README.md

```diff
@@ -0,0 +1,33 @@
+# config-migrator
+
+This is a temporary tool helping us migrating repos to build cluster(s).
+
+No image needs to be built out of this tool and we run it locally.
+
+## Steps for migrating repos
+
+* modify the `Migrated(org, repo string)` function to allow the repos that should be migrated.
+
+```bash
+### handle the non-prowgen jobs at migration: modify the find filter below
+$ release_repo=/path_to_release_repo bash -c 'find ${release_repo}  -name "*openshift-origin-master-presubmits.yaml" -exec python3 hack/migrate_jobs.py {} \;'
+### git add . then commit and push
+$ go install ./cmd/config-migrater/
```

Comment:
nit: proper spelling would be `migrator`

Reply:
Corrected.